e2e/__snapshots__/fablo-config-hlf1.4-2orgs-1chaincode.json.test.ts.snap

```diff
@@ -920,6 +933,10 @@ services:
       - ../fabric-config/crypto-config/peerOrganizations/org2.com:/var/hyperledger/cli/crypto/
       - ../fabric-config/chaincode-packages:/var/hyperledger/cli/chaincode-packages/
       - \\"$CHAINCODES_BASE_DIR/./chaincodes/chaincode-kv-node-1.4/:/var/hyperledger/cli/chaincode1/\\"
+    depends_on:
+      - ca.org2.com
+      - peer0.org2.com
+      - peer1.org2.com
```

Comment:
do we really need depends_on and list of so many machines?
Fabric is distributed system and eventually, at some point, everything should run correctly and get in sync. So ca, orderer, peers don't need to be tightly coupled and don't need each other so much.